docs/whats-new.md

```diff
@@ -2,9 +2,80 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.3
+
+Release Date: TBD
+
+### New module: @deck.gl/carto
+
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style="max-height:200px" src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/carto-01.gif?raw=true" />
+      </td>
+      <td>
+        <img style="max-height:200px" src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/carto-02.gif?raw=true" />
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+Integration with the powerful geospatial analytics platform [CARTO](https://carto.com).
+
+This new modules makes it easy to visualize regular Tables and vector tilesets in CARTO. It offers two new layers:
+
+- [CartoSQLLayer](/docs/api-reference/carto/carto-sql-layer.md): a layer to visualize data hosted in your CARTO account and to apply custom SQL.
+- [CartoBQTilerLayer](/docs/api-reference/carto/carto-bqtiler-layer.md): a layer to visualize large datasets (millions or billions of rows) directly from [Google BigQuery](https://cloud.google.com/bigquery).
+
+This effort enables applications to leverage CARTO's smart back-end capabilities, dynamic queries and much more. Visit the module's [documentation](/docs/api-reference/carto/overview.md) to get started.
+
+### TileLayer
+
+- `getTileData` now receives an additional argument `signal` that implements [AbortSignal](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal). Tile requests may now be aborted if there are too many queued or ongoing requests.
+- The `getTileData` function passed in to the current layer instance will be called rather than the first layer instance. This allows applications to progressively adjust loading behavior without having to reload all tiles.
+- New `onTileUnload` callback
+- URL templates now support `{-y}` for TMS indexing
+- Experimental support for `GlobeView` in `TileLayer` and `MVTLayer`
+- Improved memory usage in tile caching
+
+### General Performance
+
+Layers with `pickable` disabled (the default) skip generating picking attributes. This can greatly improve initial loading time and memory usage if a large layer does not have to be interactive.
+
+If no layer is pickable, deck no longer creates the picking FrameBuffer to save memory.
+
+`Deck` also added some experimental [optimization options](/docs/developer-guide/tips-and-tricks.md#optimization-for-mobile) to allow more fine-grained control of memory usage on memory-restricted devices.
+
+### pydeck 0.5
+
+pydeck 0.5 includes new event handlers — `on_hover`, `on_click`, `on_view_state_change`, and `on_resize` — which can be used to create rich interactive applications in Jupyter. This release works in sync with deck.gl 8.3.
```

Comment:
Link to https://pydeck.gl/CHANGELOG.html 

@ajduberstein are you republishing the pydeck docs?

Reply:
At the moment the best changelog we can currently link to is probably [the source of the docs](https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/docs/CHANGELOG.rst#050b1---aug-23-2020). The gif I've provided for this is from the JupyterCon demo, which is [here](https://github.com/ajduberstein/jupytercon2020/blob/master/Isochrones%20with%20GraphHopper%20and%20pydeck.ipynb), and we could link to that as well.

This needs a dedicated documentation page–I'm writing one and will have it out in the next 2-3 days as I publish pydeck 0.5. I'll update https://pydeck.gl after that.

![IMG_7734](https://user-images.githubusercontent.com/2204757/95525104-11934b00-0988-11eb-8163-1bbfe19d629d.gif)